pybamm/parameters/lithium_ion_parameters.py

```diff
@@ -641,6 +649,33 @@ def U(self, sto, T, lithiation=None):
             out.print_name = r"U_\mathrm{p}(c^\mathrm{surf}_\mathrm{s,p}, T)"
         return out
 
+    def hysteresis(self, sto):
```

Comment:
Since different hysteresis models handle this differently, I wonder if this should just be calculated from the lith and delith OCPs within the submodel?

Reply:
I believe Valentine was speaking to this item in [this](https://github.com/pybamm-team/PyBaMM/pull/3593#discussion_r1588319590) comment. Just let me know if you think that keeping it in the subclass is the best way to do this.

Reply:
Yes, let's move this into the submodel in this case

Reply:
Done.